src/app/clusters/microwave-oven-control-server/microwave-oven-control-server.h

```diff
@@ -125,28 +134,56 @@ class Delegate
     virtual ~Delegate() = default;
 
     /**
-     *   @brief Handle Command Callback in application: SetCookingParameters
-     *   @return Returns the Interaction Model status code which was user determined in the business logic
-     *   @param  cookMode: the user defined modes which from the Microwave Oven Mode application level
-     *   @param  cookTime: the input cook time value
-     *   @param  powerSetting: the input power setting value
+     *   @brief Handle Command Callback in application: SetCookingParameters.
+     *   @return Returns the Interaction Model status code which was user determined in the business logic.
+     *   @param  cookMode: the user defined modes which from the Microwave Oven Mode application level.
+     *   @param  cookTime: the input cook time value.
+     *   @param  startAfterSetting: the flag to indicate if start for the operation after handling the command.
```

Comment:
```suggestion
     *   @param  startAfterSetting: If true, cooking should start after handling the command.
```
If that's not the right summary, please reword this to describe what _is_ the right summary.